Scripts/RemediationScripts/Remediate-DontUseNonADIdentities.ps1

```diff
@@ -319,6 +333,25 @@ function Remove-AzTSNonADIdentities
     }
 
     $externalAccountsRoleAssignments = @($distinctRoleAssignmentList | Where-Object { $GuestAccountsObjectId -contains $_.ObjectId })
+    }
```

Comment:
Indentation of the block ending here needs to be fixed.

Reply:
Indentation corrected.